### ISSUE TITLE
<br> tag escaping after updating the Pods field value

### ISSUE THREAD

**Issue by raman2572** [pods-framework/pods#5049]
**Describe the bug**
I am using WYSIWYG (Visual Editor) as Field Type. Enter key doesn't work to break lines which is itself a bug.

I found a workaround by using <br> tag in the html of field. It works but once I update data in the custom post, the <br> tag is escaped from the post.

**To Reproduce**
1. Create a pod
2. Create a custom field
3. Select WYSIWYG (Visual Editor)
4. Write a sentence and use <br> to break a line
5. Publish
6. Come back to the post and edit any value
7. <br> tags will be escaped

**Expected behavior**
"Enter" should work make a break a in the sentence and <br> tag shouldn't be escaped.

## WordPress Environment

<details>
```
WordPress Version: 4.9.6

PHP Version: 5.6.36-1+ubuntu16.04.1+deb.sury.org+1

MySQL Version: 5.5.5

Server Software: nginx/1.14.0

Your User Agent: Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36

Session Save Path: /var/lib/php/sessions

Session Save Path Exists: Yes

Session Save Path Writeable: Yes

Session Max Lifetime: 1440

Opcode Cache:

Apc: No
Memcached: No
OPcache: Yes
Redis: No
Object Cache:

APC: No
APCu: No
Memcache: Yes
Memcached: Yes
Redis: Yes
WPDB Prefix: wp_

WP Multisite Mode: No

WP Memory Limit: 40M

Pods Network-Wide Activated: No

Pods Install Location: /var/www/testpanda.com/htdocs/wp-content/plugins/pods/

Pods Tableless Mode Activated: No

Pods Light Mode Activated: No

Currently Active Theme: GeneratePress

Currently Active Plugins:

All-in-One WP Migration: 6.70
Autoptimize: 2.3.4
CSV Importer For Pods: 0.8
Enhanced Category Pages: 2.3.0
GP Premium: 1.6.2
Nginx Helper: 1.9.11
Pods - Custom Content Types and Fields: 2.7.6
SEO Redirection: 4.13
Simple Custom CSS and JS: 3.17
Sucuri Security - Auditing, Malware Scanner and Hardening: 1.8.17
WP-KaTeX: 1.8.3
WP Fastest Cache: 0.8.8.1
Yoast SEO: 7.7.1
```
</details>

**Comment by jimtrue** [comment]
@raman2572 What do you have configured in Additional Options for the WYSIWYG field?

Enter will create a paragraph and a shift-enter will create a 'break' return. This is working as expected. If you type <br> in the non-visual editor, this will be interpreted as a soft break and it's immediately translated to that if you flip back to the visual editor. I'm testing this in both the regular WordPress Editor and the TinyMCE editor and they're both working identically, including adding the breaks that should be added.

Can you perhaps provide a screen recording showing the issue you're having?

**Comment by raman2572** [comment]
Hello Jim,
I am using default Wordpress editor. Output options are unchecked. 
![87e83d7e-d45b-447b-8b23-4b2fd1a7550f](https://user-images.githubusercontent.com/5188392/42104092-fc3d0b80-7be8-11e8-9213-02e51348eac8.jpeg)
![2d9028f4-494d-4e9e-a1d2-588f12bf230d](https://user-images.githubusercontent.com/5188392/42104094-fd140478-7be8-11e8-9b8a-1da1e6b10d4f.jpeg)
![e7b1a802-d876-4005-94b0-da25ac732c6f](https://user-images.githubusercontent.com/5188392/42104098-fdb0d578-7be8-11e8-83ae-e9dd0bdebb21.jpeg)

**Comment by raman2572** [comment]
Thanks it’s working after enabling wpautop

**Comment by quasel** [comment]
That's great - just to make sure there isn't a bug hiding - the default settings normally are

![screenshot 2018-07-01 11 25 49](https://user-images.githubusercontent.com/9464217/42134756-e7820538-7d41-11e8-8838-f2268a0b9597.png)

Should we add a warning what happens if they are unchecked?

**Comment by raman2572** [comment]
Yes there should be warning or a button to make default settings.

**Comment by jimtrue** [comment]
Actually, tooltips with links to the Codex for each of those would probably be adequate.

**Comment by GaryJones** [comment]
Please consider linking to The Developer reference (e.g. https://developer.wordpress.org/reference/functions/wptexturize/) instead of the Codex.

**Comment by jimtrue** [comment]
Agreed, @GaryJones though not sure why WordPress doesn't do that itself, y'know?

**Comment by jimtrue** [comment]
@pglewis added a Tasklist above
